tests/unit/test.program.js

```diff
@@ -651,7 +651,7 @@ describe('program.main', () => {
       return configObject;
     });
 
-    await execProgram(['lint', '--config', 'path/to/web-ext-config.js'], {
+    await execProgram(['lint', '--config', 'path/to/.web-ext-config.js'], {
```

Comment:
I don't think this needs to be changed, a user of web-ext can still decide to use a config file without a dot.

Reply:
Sure. User can use any config file. It is just a example